spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/aggregation/Aggregation.java

```diff
@@ -292,6 +293,19 @@ public static Fields bind(String name, String target) {
 	}
 
 	/**
+	 * Creates a new {@link GeoNearOperation} instance from the given {@link NearQuery} and the distanceField. The
+	 * {@code distanceField} defines output field that contains the calculated distance.
+	 * 
+	 * @param query must not be {@literal null}.
+	 * @param distanceField must not be {@literal null} or empty.
+	 * @return
+	 * @since 1.7
+	 */
+	public static GeoNearOperation geoNear(NearQuery query, String distanceField) {
```

Comment:
Should we actually add an overload without the distance field applied so that basically people can mimic what they got when using `new GeoNearOperation(nearQuery)` before?